apps/cookbook/src/app/guides/chart-config-guide/chart-config-guide.component.html

```diff
@@ -0,0 +1,129 @@
+<h1>Chart config</h1>
+<p>
+  The Kirby charts are based on a config strategy. The aim of this is to give the implementer full
+  control of the chart. Kirby simply provides some configurations which can be injected into a
+  <a href="https://www.chartjs.org/">ChartJS</a> chart.
+  <br />
+  <br />
+  To see more showcases on how to implement charts, please click
+  <a routerLink="../../showcase/chart-config">here</a>.
+</p>
+
+<h3>Structure</h3>
+<p>
+  Each chart type (line, stock, bar, pie) is served through a Kirby chart config class which is
+  inherited from a base class
+  <code
+    ><a
+      href="https://github.com/kirbydesign/designsystem/blob/develop/libs/designsystem/src/lib/components/charts/chart-config/chart-base-config.ts"
+      >ChartBaseConfig</a
+    ></code
+  >:
+  <br />
+  <code>
+    <a
+      href="https://github.com/kirbydesign/designsystem/blob/develop/libs/designsystem/src/lib/components/charts/chart-config/chart-stock-config/chart-stock-config.ts"
+      >StockChartConfig</a
+    ></code
+  >
+  <br />
+  <code><a href="">BarChartConfig</a> // Not yet implemented </code>
+</p>
+
+<p>In this guide we'll implement the following chart:</p>
+<kirby-card [hasPadding]="true">
+  <cookbook-chart-example-config-basic-stock></cookbook-chart-example-config-basic-stock>
+</kirby-card>
+<br />
+
+<h3>Getting started</h3>
+
+<p>To implement a Kirby chart, you'll need three things:</p>
+<ul>
+  <li><a href="https://www.chartjs.org/">ChartJS</a></li>
+  <li>
+    A HTML <a href="https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API">canvas</a> element
+  </li>
+  <li>Kirby chart config</li>
+</ul>
+
+<p>In your component template, add a canvas to your HTML and give it an <code>id</code>:</p>
+
+<cookbook-code-viewer
+  [html]="guideChartHtml"
+  [language]="'html'"
+  [inlineLabel]="true"
+></cookbook-code-viewer>
+<p>
+  The canvas must be wrapped in an element with <code>position: relative</code> in order to be
+  responsive. A thorough explanation can be found
+  <a href="https://www.chartjs.org/docs/latest/configuration/responsive.html#important-note"
+    >here</a
+  >
```

Comment:
```suggestion
  >.
```
Mr. Mitpick at your service 😬